docs: various (minor) changes and rewording (#3661)
docker/cli: docs/reference/commandline/pull.md

can contain multiple images. To pull all images from a repository, provide the
`-a` (or `--all-tags`) option when using `docker pull`.

This command pulls all images from the `ubuntu` repository:

```console
$ docker image pull --all-tags ubuntu

Pulling repository ubuntu
ad57ef8d78d7: Download complete
105182bb5e8b: Download complete
511136ea3c5a: Download complete
73bd853d2ea5: Download complete
....

Status: Downloaded newer image for ubuntu
```

> After the pull has completed use the `docker image ls` (or `docker images` shorthand)
> command to see the images that were pulled. The example below shows all the `ubuntu`
> images that are present locally:

Reviewer: ```suggestion
After the pull has completed, use the `docker image ls` command (or the `docker images` shorthand) to see the images that were pulled. The example below shows all the `ubuntu`
images that are present locally:
```

running in a terminal, will terminate the pull operation.

```console
$ docker pull ubuntu

Using default tag: latest
latest: Pulling from library/ubuntu
a3ed95caeb02: Pulling fs layer
236608c7b546: Pulling fs layer
^C
```

> The Engine terminates a pull operation when the connection between the Docker
> Engine daemon and the Docker Engine client initiating the pull is lost. If the
> connection with the Engine daemon is lost for other reasons than a manual
> interaction, the pull is also aborted.

Reviewer: The formulation is bit unclear. I would formulate it like the pull is aborted when: 1) connection between daemon and client is cut or lost or the operation is manually terminated.

```suggestion
The Engine terminates a pull operation when the connection between the daemon and the client (initiating the pull) is cut or lost for any reason or the command is manually terminated. 
```
